paper-checkbox.js

```diff
@@ -85,12 +85,16 @@ const template = html`<style>
     pointer-events: none;
     -webkit-transition: background-color 140ms, border-color 140ms;
     transition: background-color 140ms, border-color 140ms;
+
+    @apply --paper-transition-easing;
```

Comment:
I don't think this is the right name for this mixin because it can potentially be used to change any property of elements this ruleset applies to, which may not be related to the transition. If you're interested in `transition-timing-function` specifically, I'd recommend adding a single CSS custom property:
```css
-webkit-transition-timing-function: var(--paper-checkbox-transition-timing-function, initial);
transition-timing-function: var(--paper-checkbox-transition-timing-function, initial);
```

What properties are you looking to change?

Reply:
This name is widely used in other components in paper-elements like:
https://github.com/PolymerElements/paper-input/blob/master/paper-input-container.js#L214

It is mostly used for overwriting transform and animation time to 0, mostly in scuba tests.

Reply:
> This name is widely used in other components in ...

Could you show me other examples? Here are two examples of using individual CSS custom properties: [iron-collapse](https://github.com/PolymerElements/iron-collapse/blob/ba1bdfba0e63dd526377c8210988fd58a5772f07/iron-collapse.js#L68-L70) and [paper-progress](https://github.com/PolymerElements/paper-progress/blob/d18eb7acf941640068ea6c8d8c6e0aa9ef1b4e8f/paper-progress.js#L134-L144) ([2](https://github.com/PolymerElements/paper-progress/blob/d18eb7acf941640068ea6c8d8c6e0aa9ef1b4e8f/paper-progress.js#L173-L187)).

Reply:
Sorry, I guess I was tracking through multiple paper components like dropdown but it eventually all come down to paper-input...

I will try more specified mixin